Rakefile

```diff
@@ -2,28 +2,37 @@
 
 require 'html-proofer'
 
-task :default => [:test]
+task default: :test
 
 desc 'Runs the tests!'
-task :test => :build do
+task test: :build do
   Rake::Task['run_html_proofer'].invoke
 end
 
 desc 'Builds the site'
-task :build => [:remove_output_dir, :gen_trusty_image_data, :gen_gce_ip_addr_range] do
-  FileUtils.rm '.jekyll-metadata' if File.exist?('.jekyll-metadata')
+task build: %i[remove_output_dir regen] do
+  rm_f '.jekyll-metadata'
   sh 'bundle exec jekyll build --config=_config.yml'
 end
 
 desc 'Remove the output dir'
 task :remove_output_dir do
-  FileUtils.rm_r('_site') if File.exist?('_site')
+  rm_rf('_site')
 end
 
 def print_line_containing(file, str)
   File.open(file).grep(/#{str}/).each do |line| puts "#{file}: #{line}" end
 end
 
+def dns_txt(hostname)
+  require 'faraday'
```

Comment:
Is it a normal ruby pattern to have `require`s here?

Reply:
I would say it is not.

Technically it is a small optimization if this method is not invoked, but it seems to me that these libraries are commonly used to the point where frequent `require`s are more nuisance.

Reply:
My perspective is that ruby authoring in the context of a Rakefile is a sub-genre of its own, where deferring requirements like this is much more common and a generally good idea.... _but_ in _this case_, I fully admit that my tendency to defer requirements is overkill given the typical commands that are run. :grin: